src/Nethermind/Nethermind.Runner/Program.cs

```diff
@@ -140,9 +140,10 @@ private static void Run(string[] args)
                     .OrderBy(p => p.Name))
                 {
                     ConfigItemAttribute? configItemAttribute = propertyInfo.GetCustomAttribute<ConfigItemAttribute>();
-                    if (!(configItemAttribute?.HiddenFromDocs ?? false))
+                    if (!(typeLevel?.DisabledForCli ?? true))
```

Comment:
shouldn't it be: if (!(configItemAttribute?.DisabledForCli ?? false))
1. You are using top level twice
2. The default value is true